x-pack/plugins/actions/server/usage/actions_usage_collector.ts

```diff
@@ -37,8 +39,10 @@ export function createActionsUsageCollector(
       return true;
     },
     schema: {
+      alert_history_connector_enabled: { type: 'boolean' },
       count_total: { type: 'long' },
       count_active_total: { type: 'long' },
+      count_active_alert_history_connectors: { type: 'long' },
```

Comment:
Do you mind adding the descriptions to the fields. We are trying to enforce that, at least the new fields must be documented. 

You can do so by adding something like

```
count_active_alert_history_connectors: { type: 'long', _meta: { description: 'Number of active alerts...' } },
```

Thank you!

Reply:
@afharo Thank you for reviewing! Descriptions have been added in this commit: https://github.com/elastic/kibana/pull/97063/commits/3b41bbf14a1d0d1cbfd1963d5de3d93ac7511f0b